src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,35 @@
+.calendar {
+  margin: 0 auto;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.